Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN rm -rf work
 RUN pip install docutils==0.17.1 # Need to pin docutils to an old version for now, due to https://github.com/executablebooks/jupyter-book/issues/2022
 RUN pip install referencing
 RUN pip install jupyter-book
-RUN pip install numpy jinja2 altair_data_server vl-convert-python click ibis-framework ghp-import jupytext nodejs
+# Pinning pandas until altair 5.1.2 to avoid future warning https://github.com/altair-viz/altair/issues/3181
+RUN pip install numpy jinja2 pandas"<2.1" altair">=5.1.1" "vegafusion[embed]" vl-convert-python">=0.13" click ibis-framework ghp-import jupytext nodejs
```

Comment:
Although I added these here, it seems that I still need `pip install ...` in the build files. I thought the docker file would automatically update the container, but maybe there is a manual step involved to update the docker file?

Reply:
@joelostblom the dockerfile only rebuilds when commiting to `main` (otherwise we would have to do more gymnastics with temporary docker images etc. i'll open an issue about it, but for now I'll merge `main` into this branch to get the new updated dockerfile